identity/README.md

```diff
@@ -53,10 +40,12 @@ With these in place, you'll be able to browse Identity on
 ## Configuration of local Identity API
 
 The Identity site can be configured to use the local Identity API with the
-following properties in `frontend.properties`.
+following properties in `~/.gu/frontend.conf`.
 
 ```
 # ID
```

Comment:
minor suggestion, but I'd put the id comment inside devOverrides, as the person might have other overrides that apply to other things.